setup.md

```diff
@@ -5,33 +5,27 @@ title: Setup
 
 ## Getting ready
 
-You need to download and install OpenRefine and download a data file to follow this lesson.
+You need to install OpenRefine and download a data file to follow this lesson.
 
-### Downloading OpenRefine
+### Installing and running OpenRefine
 
-You can download OpenRefine from [http://openrefine.org/download.html](http://openrefine.org/download.html). This lesson has been tested with all versions of OpenRefine up to the latest tested version, 3.1.  
+You can download OpenRefine from [http://openrefine.org/download.html](http://openrefine.org/download.html). This lesson has been tested with all versions of OpenRefine up to the latest tested version, 3.1.
 
 If you are using an older version, it is recommended you upgrade to the latest tested version. 
 
 There are versions for Windows, Mac OS X and Linux.
 
-### Installing and Running OpenRefine
+Please follow the installation instructions on the OpenRefine wiki: [Installation Instructions](https://github.com/OpenRefine/OpenRefine/wiki/Installation-Instructions)
 
-When you download OpenRefine for Windows or Linux from the address above, you are downloading a zip file. To install OpenRefine you simply unzip the downloaded file wherever you want to install the program. This can be to a personal directory or to an applications or software directory - OpenRefine should run wherever you put the unzipped folder. The location has to be a "local" drive as problems have been reported trying to run OpenRefine from a Network drive.
+Notes:
```

Comment:
I don't think the installation instructions on the main wiki mention issues with running OpenRefine from a Network drive. I've found this particularly important to mention for users working in an institutional context. I think we should keep this brief paragraph on the installation here, while adding a link to the installation instructions on the OpenRefine wiki for additional detail

Reply:
Thanks Owen for the detailed review. I have restored the paragraph on the installation as you recommended.